lib/ziti-tunnel-cbs/ziti_tunnel_cbs.c

```diff
@@ -398,7 +398,7 @@ ziti_intercept_t *new_ziti_intercept(ziti_context ztx, ziti_service *service, zi
     for (int i = 0; i < sizeof(intercept_cfgtypes) / sizeof(cfgtype_desc_t); i++) {
         cfgtype_desc_t *cfgtype = &intercept_cfgtypes[i];
         const char *cfg_json = ziti_service_get_raw_config(service, cfgtype->name);
-        if (cfg_json != 0 && cfgtype->parse(&zi_ctx->cfg, cfg_json, strlen(cfg_json)) == 0) {
+        if (cfg_json != 0 && cfgtype->parse(&zi_ctx->cfg, cfg_json, strlen(cfg_json)) > 0) {
```

Comment:
Should this test be `>=` ?

Reply:
This line must be appearing in my PR because I merged main branch to one of my PRs. I dont think I modified this line